content/docs/reference/self-encryption-key.md

```diff
@@ -1,16 +0,0 @@
----
```

Comment:
I would keep this file, it doesn't have to be detailed, content can be similar or copy/pasted from what is contained elsewhere in the reference section. It is useful for reference definitions to be short and sweet so that we can link to them from other content pages.